tests/ClientTest.php

```diff
@@ -202,14 +202,16 @@ public function testSendChecksBeforeSendOption(): void
         $transport->expects($this->never())
             ->method('send');
 
-        $client = ClientBuilder::create([
-            'dsn' => 'http://public:secret@example.com/1',
-            'before_send' => function () use (&$beforeSendCalled) {
-                $beforeSendCalled = true;
+        $options = new Options(['dsn' => 'http://public:secret@example.com/1']);
```

Comment:
Instead of instantiating the `Options` object I would prefer to stick with the plain `array` of options